### ISSUE TITLE
calling get_correlation_ids exception occurs

### ISSUE THREAD

**Issue by Shikkic** [DataDog/dd-trace-py#851]
I just setup injecting `trace_id` and `span_id` into a custom logger and it works well for most logs however in certain circumstances (I think for example a request that throws an exception right away) the function `get_correlation_ids` throw an exception:

```
AttributeError: 'NoneType' object has no attribute 'get_current_span'
return self.get_call_context().get_current_span()
File "/usr/local/lib/python3.7/site-packages/ddtrace/tracer.py", line 310, in current_span
span = tracer.current_span()
File "/usr/local/lib/python3.7/site-packages/ddtrace/helpers.py", line 34, in get_correlation_ids
trace_id, span_id = helpers.get_correlation_ids()
```

I wonder if this is because in `get_correlation_ids` [it always assumes the the global `tracer` is initialized and not `None`](https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/helpers.py#L28). That's the only way to generate this exception from within `get_correlation_ids`. So my question is, is that possible and something that just wasn't accounted for and should be checked?

Using dd-trace-py version `0.22.0`

**Comment by brettlangdon** [comment]
Fixed by #852 and will go out in the next release.

If you wanted to give it a try until then you can install from the next release branch via:

```
pip install git+https://github.com/datadog/dd-trace-py@0.24-dev
```

**Comment by Shikkic** [comment]
Thanks @brettlangdon, I'll give it a try on the new release and can reopen if I notice the same behavior happening. I appreciate your quick response and work to help fix this!
